semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/package.scala

```diff
@@ -132,6 +132,10 @@ package object language {
   implicit def toComment(steps: Steps[nodes.Comment]): Comment =
     new Comment(steps.raw)
 
+  implicit def toLineNumberEndAccessors[A <: StoredNode with HasLineNumberEnd](
```

Comment:
Is there any reason we choose implicit conversions over implicit classes?

Reply:
Dont know whether this was Micheal motivation but using the implicit methods allows you to split the class definition from the implicit conversion functionality which lets you group the implicit stuff in common places like we do with `io.shiftleft.semanticcpg.language._`.

Reply:
Yes exactly, the main idea is to have the implicit conversions in few places, so that in ocular/joern we just use the `language._` import and have everything at the user's fingertips. That pattern is used across all of our modules. 

That's not to say that implicit classes are bad, but it's already quite a burden to keep our downstream projects in sync, and with the implicits scattered in more places we'd have a harder time maintaining ocular/joern imports. Makes sense?